src/@types/inews/index.d.ts

```diff
@@ -19,7 +19,7 @@ declare module 'inews' {
 		 * 	but may be used in blueprints - installation / organisation dependent.
 		 */
 		interface INewsFields {
```

Comment:
![image](https://user-images.githubusercontent.com/6236874/141115972-94d6dbf3-93cc-4e30-90e9-bb7a4154cdc1.png)

Reading where the fields-object comes from, the parser in the iNews FTP library, we cant guarantee any of the keys nor objects: https://github.com/SuperFlyTV/node-inews/blob/3ef22df660da7f5d3100676a177da54607e8645a/inewsStoryParser.js#L115

Possible improvements

- Loose types to indicate that we cant rely on any key or any value
- Strict parsing in the underlying parsing
- A 2-step process where raw/unsafe data comes from the underlying parser and then a strict parser checks against som strict types here: https://github.com/tv2/inews-ftp-gateway/blob/a39692166a26de32a3a5c4e77e28df465847cddd/src/classes/RundownManager.ts#L109